code/modules/vehicles/unmanned/unmanned_droid.dm

```diff
@@ -13,6 +13,8 @@
 	var/datum/supply_beacon/beacon_datum
 	/// Action to activate suppply antenna. 
 	var/datum/action/antenna/antenna
+	/// The mob controlling the droid remotely
+	var/mob/living/remote_user
```

Comment:
I wonder if this should be harddel protected.

I say it is a good place to use weakref

Reply:
Yea should be harddel protected imo